views/mixins.jade

```diff
@@ -16,11 +16,11 @@ mixin renderImage(image, slug, w)
   if image && image.fields.file
     if slug != ""
       a(href="/products/"+slug data-nav) 
-        img(src=image.fields.file.url alt=image.fields.title width=w)
+        img(src=image.fields.file.url+"?w="+w alt=image.fields.title width=w)
```

Comment:
Hmm,

http://webmasters.stackexchange.com/questions/68491/specifying-image-dimensions-to-improve-browser-performance

Maybe the width argument should still stay?